source/dlangbot/app.d

```diff
@@ -43,7 +43,7 @@ void startServer(HTTPServerSettings settings)
         .get("/", (req, res) => res.render!"index.dt")
         .get("*", serveStaticFiles("public"))
         .post("/github_hook", &githubHook)
-        .match(HTTPMethod.HEAD, "/trello_hook", (req, res) => res.writeVoidBody)
+        .match(HTTPMethod.HEAD, "/trello_hook", (HTTPServerRequest req, HTTPServerResponse res) => res.writeVoidBody)
```

Comment:
Reported and filed issue + PR upstream: https://github.com/rejectedsoftware/vibe.d/pull/1866, https://github.com/rejectedsoftware/vibe.d/pull/1867